locations/spiders/suzuki_marine_au.py

```diff
@@ -0,0 +1,30 @@
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.categories import Categories
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class SuzukiMarineAUSpider(Spider):
+    name = "suzuki_marine_au"
+    item_attributes = {"brand": "Suzuki", "brand_wikidata": "Q181642", "extras": Categories.SHOP_BOAT.value}
+
+    def start_requests(self):
+        yield JsonRequest(url="https://www.suzukimarine.com.au/find-dealers/dealersByState?state=all")
+
+    def parse(self, response, **kwargs):
+        for feature in response.json():
```

Comment:
I think we should also probably check `Services` in the source data which can be a value of `0`, `1` or `2` which would determine whether the location is `shop=boat` (you can buy boats or boat parts), `shop=boat_repair` (you can mostly or only service/repair your boat but not buy boats or boat parts) or `shop=boat` + `boat:repair=yes` if you can buy boats and boat parts, and also service/repair boats.

Another approach ATP has used is the `mazda_au` spider for an example of where up to 3 features are extracted per car dealership: a car shop, a car parts shop and a car repair/service workshop. I suppose this same approach could be used for boats. Or spiders like `mazda_au` should have tags changed to be `shop=car` + https://wiki.openstreetmap.org/wiki/Key:service:vehicle:car_repair + https://wiki.openstreetmap.org/wiki/Key:service:vehicle:car_parts so that only one feature is ever extracted for each dealership.

@Cj-Malone @matkoniecz as OSM tagging experts, any preferences for multiple feature extraction or only having a single feature per car/boat dealership that uses a `shop=car` or `shop=boat` primary tag, and then adds part sales and repair/service tags as additional tags as appropriate.

Reply:
Variations:
"Services":"[\"0\",\"1\",\"2\"]",
"Services":"[\"0\",\"1\"]",
"Services":"[\"0\"]",
"Services":"[\"2\"]",

Reply:
Well.... it's not immediately obvious which is which.

Reply:
I guess it depends. For example if we know that car repair and sale is integrated into one POI then splitting it makes no sense.

But of fuel station and its shop are treated as separate (at least in OSM they are) then splitting makes sense.

I have no on the ground knowledge here.